$MirOS: ports/devel/sdl-ttf/patches/patch-SDL_ttf_c,v 1.1 2006/06/24 17:41:42 bsiegert Exp $
--- SDL_ttf.c.orig	Thu Dec 16 05:39:32 2004
+++ SDL_ttf.c	Thu Jun 29 17:25:45 2006
@@ -43,17 +43,7 @@
 #include FT_FREETYPE_H
 #include FT_OUTLINE_H
 #include FT_TRUETYPE_IDS_H
-/*
-#include <freetype/freetype.h>
-#include <freetype/ftoutln.h>
-#include <freetype/ttnameid.h>
-*/
-#include <freetype/internal/ftobjs.h>
 
-#ifndef FT_OPEN_STREAM
-#define FT_OPEN_STREAM ft_open_stream
-#endif
-
 #include "SDL.h"
 #include "SDL_endian.h"
 #include "SDL_ttf.h"
@@ -275,7 +265,7 @@ TTF_Font* TTF_OpenFontIndexRW( SDL_RWops
 	}
 	memset(stream, 0, sizeof(*stream));
 
-	stream->memory = library->memory;
+	stream->memory = NULL;  /* set by FT_Open_Face */
 	stream->read = RWread;
 	stream->descriptor.pointer = src;
 	stream->pos = (unsigned long)position;
